cfme/tests/containers/test_basic_metrics.py

```diff
@@ -1,19 +1,17 @@
 import pytest
+import re
 from utils import testgen
 from utils import conf
 from utils.version import current_version
 
 
 pytestmark = [
     pytest.mark.uncollectif(lambda provider: current_version() < "5.6"),
-    pytest.mark.usefixtures('setup_provider'),
```

Comment:
I think you'll still need to setup the provider? No? Else what's the point of having the provider. Also what's the purpose of this test, it doesn't interact with ManageIQ at all.